### ISSUE TITLE
Sign-up overlay, notifications, & loading bar must be inside a page landmark

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2402]
These should be children of `<header>` or `<main>`.

<img width="566" alt="Screen Shot 2019-10-17 at 4 55 42 PM" src="https://user-images.githubusercontent.com/12454203/67055962-0738cb00-f0ff-11e9-8171-8eec8b7fc05f.png">

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_

**Comment by Santoz2** [comment]
Accessibility Triage Notes: UW ATX considers this a priority 3 issue. It is true that, in general, all content should be contained inside of landmarks, because landmarks are a primary form of navigation for many users.

However, this issue presents 3 different edge cases. None of these 3 are typically content that users would use landmarks to navigate to. If these widgets behave as they should (which, I believe is discussed in other issues on this GitHub repo), then adding these elements to landmarks would be still good to do, but would not affect user behaviors much.

One thing I noted, though, while going through this issue, is that landmarks can be used to jump from the sign in page to the page behind it.

Interestingly, this example actually contains a solution in which the modal is *not* contained in a landmark: [https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html)

It adds the style overflow: hidden to the body of the page, so that the rest of the content cannot be reached while the user is on the modal. Another example adds aria-hidden to the content in the page not contained in the modal to hide the rest of the page.

**Comment by dananjohnson** [comment]
Thanks for your feedback, @Santoz2. We are going to close this issue for now. It would a fairly significant refactor to address it, and given the low impact your team feels it has on user experience, it doesn't seem like a worthwhile undertaking at this time. We can reopen the issue should this become a higher priority at a later date.

Re: the sign-in page issue, a fix is in place in PR #3079!
